src/index.js

```diff
@@ -18,16 +18,17 @@ const logger = require('@adobe/aio-lib-core-logging')('aio-lib-analytics', { lev
 /**
 * Returns a Promise that resolves with a new AnalyticsCoreAPI object.
 *
-* @param companyId {string} company ID to be used with Adobe Analytics.
+* @param companyId {string} company ID to be used with Adobe Analytics.  If this is not available, it should be set to null and the imsOrgId included in the parameters.
 * @param apiKey {string} Your api key
 * @param token {string} Valid auth token
+* @param imsOrgId {string} Valid IMS Org ID (i.e., 'F9452D1224FEA71029B5C8A97@AdobeOrg' )  Only passed when a companyId is unavaiable.
 * @returns {Promise<AnalyticsCoreAPI>}
 */
-function init (companyId, apiKey, token) {
+function init (companyId, apiKey, token, imsOrgId) {
```

Comment:
Instead of adding imsOrgId to init params we can add an options param `(companyId, apiKey, token, options )`
and let imsOrgId be specified as key to this options object. This makes sure any future changes to init won't change function signature and can easily be accommodated in options object.